integration-tests/hibernate-tenancy/src/main/resources/application.properties

```diff
@@ -0,0 +1,38 @@
+# Hibernate ORM settings 
+quarkus.hibernate-orm.database.generation=none
+quarkus.hibernate-orm.multitenant=DATABASE
+quarkus.hibernate-orm.validate-tenant-in-current-sessions=false
```

Comment:
this property no longer exists

Reply:
Fixed with https://github.com/quarkusio/quarkus/pull/9423